values/gitea/gitea.gotmpl

```diff
@@ -48,21 +49,52 @@ gitea:
     username: {{ $g.adminUsername }}
     password: {{ $g.adminPassword }}
   config:
+    admin:
+      DISABLE_REGULAR_ORG_CREATION: false
     log:
       LEVEL: trace
+    {{ with $v | get "smtp" nil }}
+    mailer:
+      enabled: true
+      HELO_HOSTNAME: {{ .hello }}
+      FROM: {{ .from }}
+      USER: {{ .auth_username }}
+      PASSWD: {{ .auth_password }}
+      MAILER_TYPE: smtp
+      IS_TLS_ENABLED: true
+      SUBJECT_PREFIX: 'Otomi[Gitea]: '
+    {{- end }}
     openid:
       ENABLE_OPENID_SIGNIN: true
       ENABLE_OPENID_SIGNUP: true
+    oauth2_client:
+      OPENID_CONNECT_SCOPES: openid email profile
```

Comment:
how about groups claim?

Reply:
??

Reply:
Like I mentioned somewhere else, keycloak populates it by default